### PR TITLE
[ticket/13523] Remove PHP 5.2 builds from travis configuration

### PR DESCRIPTION
**Pull request by marc1706**
This is only for develop-olympus. Please make sure that this doesn't break the builds on develop-ascraeus and newer or ask me for another PR that reverts this in develop-ascraeus and newer.

Ticket: https://tracker.phpbb.com/browse/PHPBB3-13523

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 language: php
 php:
-  - 5.2
```
Comment (nickvergessen) [inline .travis.yml]: Everything in this file, apart from this change here, should be enough to fix it?

Comment (marc1706) [inline .travis.yml]: I'm not sure what you'e trying to say. Can you elaborate on this?

Comment (nickvergessen) [inline .travis.yml]: Instead of removing 5.2 tests, we should try to fix them?
Or is there no phpunit version available for 5.2 anywhere?

Comment (marc1706) [inline .travis.yml]: Please take a look at the ticket.

**Comment by bantu** [comment]
Closing this in favour of #3310

**Pull request closed without merging**